recipes/purepng/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "purepng" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "0d64d1464252184dae30c9a4611faed72868a6e352fce58019a9b4d38c3a94cd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
```

Comment:
If you are compiling a C-extension (it looks like via cython), please add `toolchain` to the `build` requirements:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33